recipes/googlesearch/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: googlesearch
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/google/google-{{ version }}.tar.gz
+  sha256: ff7d997dbf33be666ed5deb8c5abf5d103c27ec199559b6d994923469e26bb48
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
```

Comment:
You don't need this build section IIUIC.

Reply:
Since pip did not get auto-installed either (and the `setup.py` explicitly loads objects from it), I decided to be cautious/explicit. Will try removing.